web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
```

Comment:
> I agree – the "status" page is about status of scraping more than the loaded configuration. 

I disagree. I see the status page as the top-level page for finding more information about the Prometheus instance, that includes both how it's configured and how it's currently running. Having it merely as a list of links would be wasteful, nor should it be limited to just debug target/scrape information.

The current debug target/scrape information should move to a sub page, as should the config, flags, and rules as it's not sane to have these all at the top level as they can all (bar flags) get arbitrarily long.